src/xtopcom/xevm_common/src/trie/xtrie_db.cpp

```diff
@@ -319,6 +319,48 @@ xbytes_t xtop_trie_db::preimage_key(xh256_t const & hash_key) const {
     return res;
 }
 
+void xtop_trie_db::cleans_put(xh256_t const & hash, xbytes_t const & data) {
+#if defined(CACHE_SIZE_STATISTIC) || defined(CACHE_SIZE_STATISTIC_MORE_DETAIL)
```

Comment:
only need check defined CACHE_SIZE_STATISTIC

Reply:
CACHE_SIZE_STATISTIC_MORE_DETAIL is independent of CACHE_SIZE_STATISTIC